### PR TITLE
build: added 2.13.0-M2 cross build option

### PR DESCRIPTION
**Pull request by noam-almog**
upgraded scalatest 3.0.0 -> 3.0.4
updated build to cross build 2.13.0-M2

### CODE REVIEW

**Comment by pjfanning** [comment]
I tried building locally with scala 2.13.0-M3 but the compilation fails due to https://github.com/scala/bug/issues/10714

**Comment by noam-almog** [comment]
can anyone help merging this ?

**Comment by pjfanning** [comment]
@noam-almog the change is failing the travis tests

**Comment by noam-almog** [comment]
@pjfanning it's failing with ```java.lang.IllegalArgumentException: invalid source release: 1.8``` on 2.10 and 2.11 i think it's something related to travis, do you have any idea on how to resolve this ?

note, most of the libraries are dumping support for 2.10

File: build.sbt [hunk 1]
```diff
@@ -23,46 +23,58 @@ lazy val java7Home =
...
-)
-
-scalacOptions ++= (
-  if (scalaVersion.value.startsWith("2.12")) {
-    // -target is deprecated as of Scala 2.12, which uses JVM 1.8 bytecode
-    Seq.empty
+javacOptions ++= javaOptionsFor(scalaVersion.value)
+
+def javaOptionsFor(version: String) =
+  if (version.startsWith("2.12")) {
```
Comment (pjfanning) [inline build.sbt]: I think this is the change that is causing the travis build failures

**Comment by sullis** [comment]
This PR is obsolete.

**Pull request closed without merging**